src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -25,6 +25,10 @@ private StringHelper() {
         // utility class
     }
 
+    public static boolean isEmpty(String s) {
```

Comment:
Let's add a header comment here and a proper unit test for this method in `StringHelperTest`.